### ISSUE TITLE
Log in link after creating account 

### ISSUE THREAD

**Issue by AlexDM0** [crownstone/crownstone-cloud#8]
If you make an account because of an invite link, there is a login link after signing up.

This has to be improved and clear user instructions should be added.

**Comment by mrquincle** [comment]
Is this a different email then when registering?

**Comment by AlexDM0** [comment]
Unlikely, given that creating account is the same as registering

**Comment by mrquincle** [comment]
Then it already got a nice icon and clickable links. :-)

**Comment by AlexDM0** [comment]
Yes, but after you CLICK that link, there is a log in link. That should not be there
